processor/filterprocessor/logs.go

```diff
@@ -38,6 +40,22 @@ func newFilterLogsProcessor(set processor.Settings, cfg *Config) (*filterLogProc
 	}
 	flp.telemetry = fpt
 
+	if len(cfg.LogConditions) > 0 {
+		pc, collectionErr := common.NewLogParserCollection(set.TelemetrySettings, common.WithLogParser(filterottl.StandardLogFuncs()))
+		if collectionErr != nil {
+			return nil, collectionErr
+		}
+		var errors error
+		for _, cs := range cfg.LogConditions {
+			metricConsumer, parseErr := pc.ParseContextConditions(cs)
+			errors = multierr.Append(errors, parseErr)
+			flp.consumers = append(flp.consumers, metricConsumer)
+		}
+		if errors != nil {
+			return nil, errors
+		}
+	}
```

Comment:
We should early return here as `skipExpr` won't be evaluated when the signal condition is set.